core/src/main/java/org/radargun/stages/test/legacy/LegacyTestStage.java

```diff
@@ -166,7 +169,9 @@ public void run() {
             throw new IllegalStateException("Interrupted during ramp-up.", e);
          }
       }
+      
```

Comment:
It's strange that checkstyle does not complain about whitespaces at end of line...